src/api-routes/persons/{osuId}.js

```diff
@@ -0,0 +1,24 @@
+import { errorHandler, errorBuilder } from 'errors/errors';
+import { getPersonById } from '../../db/oracledb/persons-dao';
+
+/**
+ * Get person by ID
+ *
+ * @param {object} req request
+ * @param {object} res response
+ * @returns {Promise<object>} response
```

Comment:
Type of [`RequestHandler`](https://github.com/osu-mist/express-api-skeleton/blob/ab11f6dcd34ac793a0d4f4243bf29c40b20770bb/.eslintrc.yml#L40) has already defined: https://github.com/osu-mist/identities-api/blob/83d0112d7624a393b7232711d89408ce1d049dc7/src/api-routes/identities/%7BosuId%7D.js#L7